### PR TITLE
Added Release notes for v1.3.6 patch release

### PR DESCRIPTION
**Pull request by anajosep**
Release notes for 1.3.6 patch release containing fix for bug VZ-7190

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -50,6 +50,11 @@ Fixes:
 - Updated base and other images to resolve CVEs.
 
 
+### v1.3.6
+Fixes:
+
+- Resolved an issue in OpenSearch where user provided JVM Heap size setting was not used.
```
Comment (MelwynPaul) [inline content/en/docs/releasenotes/_index.md:56]: Heap > heap

**Review by MelwynPaul — COMMENTED** [review]
Looks good. 
Just one minor comment.
Thank you.

**Review by MelwynPaul — APPROVED** [review]
Looks good. 
Thank you.

**Review by lake-of-dreams — APPROVED** [review]

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -50,6 +50,11 @@ Fixes:
 - Updated base and other images to resolve CVEs.
 
 
+### v1.3.6
+Fixes:
+
+- Resolved an issue in OpenSearch where user provided JVM heap size setting was not used.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:56]: where user provided -> where the user provided

**Review by rosemarymarano — COMMENTED** [review]
At a minimum, please make this correction in the master branch.

**Pull request merged**